_includes/links.md

```diff
@@ -0,0 +1,3 @@
+[github-flavored-markdown]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet
+[gitlab-flavored-markdown]: https://docs.gitlab.com/ee/user/markdown.html
+
```

Comment:
Since we moved over to the remote theme (see #5), the `_includes/link.md` file has been removed. I think it would be good to still use that approach in this lesson, which means we need to replicate the full set of link refs from https://github.com/carpentries/carpentries-theme here. That will allow the theme to use the refs in our local `links.md` file instead of the remote equivalent. If some of that didn't make sense don't worry, the solution is simple enough: add the entire contents of [this file](https://github.com/carpentries/carpentries-theme/blob/master/_includes/links.md) to the `_includes/links.md` file in your branch.

Reply:
Thank you, I copied the links from bf45de8.